xbmc/settings/Settings.cpp

```diff
@@ -422,6 +422,7 @@ const std::string CSettings::SETTING_GENERAL_ADDONBROKENFILTER = "general.addonb
 const std::string CSettings::SETTING_SOURCE_VIDEOS = "source.videos";
 const std::string CSettings::SETTING_SOURCE_MUSIC = "source.music";
 const std::string CSettings::SETTING_SOURCE_PICTURES = "source.pictures";
+const std::string CSettings::SETTING_VIDEOSCREEN_HW_SCALING_FILTER="videoscreen.hwscalingfilter";
```

Comment:
This doesn't need to live here. Better to put it where it is used. (We have moved away from putting specific settings in CSettings)

Reply:
Yeah this will be moved appropriately in using file.